controller/eventssyncer/eventssyncer.go

```diff
@@ -72,13 +93,46 @@ func (e *EventsSyncer) convertEventToClusterEvent(event *v1.Event, cluster *clus
 		Name:        event.Name,
 		Labels:      event.Labels,
 		Annotations: event.Annotations,
+		Namespace:   ns.Name,
 	}
 	ref := metav1.OwnerReference{
-		Name:       e.clusterName,
-		UID:        cluster.UID,
-		APIVersion: cluster.APIVersion,
-		Kind:       cluster.Kind,
+		Name:       ns.Name,
```

Comment:
when a resource is namespaced in namespace FOO, what is the point of giving it an owner reference to namespace FOO?
namespace FOO already own it by virtue of it being in that namespace and if namespace FOO is removed, the resource itself will automatically be deleted

Reply:
@cjellick :) no reason, leftovers from before. Will cleanup!